src/lib/run-studies.ts

```diff
@@ -5,6 +5,36 @@ import { getECSTaskDefinition, registerECSTaskDefinition, runECSFargateTask, get
 import { managementAppGetRunnableStudiesRequest, toaGetRunsRequest, filterManagmentAppRuns } from './utils'
 import 'dotenv/config'
 
+const ecsClient = new ECSClient()
+const taggingClient = new ResourceGroupsTaggingAPIClient()
+
+// Set in IaC
+const cluster = process.env.ECS_CLUSTER || ''
+const baseTaskDefinition = process.env.BASE_TASK_DEFINITION_FAMILY || ''
+const subnets = process.env.VPC_SUBNETS || ''
+const securityGroup = process.env.SECURITY_GROUP || ''
+
+// In case we want to test stuff without connecting to mgmt app
+const _managementAppSampleData = {
```

Comment:
Do we still need the `_managementAppSampleData` ?

Reply:
Yeah, good point. I think this is pretty early test code we kept laying around for convenience which may no longer be all that useful. I went ahead and [removed it](https://github.com/safeinsights/setup-app/pull/10/commits/02c6dc72b4746976222e68576fb94db4537019d8).